tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -137,7 +137,7 @@ jobs:
     inputs:
       script: |-
         echo "Select agent: $(Agent.Name), GPU: $HIP_VISIBLE_DEVICES, render: $DRIVER_RENDER"
-        bash $(Build.SourcesDirectory)/tools/ci_build/github/pai/pai_get_thread.sh $(Agent.Name) $HIP_VISIBLE_DEVICES
+        bash $(Build.SourcesDirectory)/tools/ci_build/github/pai/pai_clean_device.sh $(Agent.Name) $HIP_VISIBLE_DEVICES
```

Comment:
If the pai_clean_device.sh failed, should the pipeline continue?

Reply:
we need stop pipeline if clean device step failed.

Reply:
Done.